tests/components/test_component_isotropic_remeshing.cpp

```diff
@@ -657,3 +658,43 @@ TEST_CASE("remeshing_with_boundary", "[components][isotropic_remeshing][2D]")
         // mesh.serialize(writer);
     }
 }
+
+TEST_CASE("remeshing_preserve_topology", "[components][isotropic_remeshing][2D]")
```

Comment:
A test case that is under development can be removed from the CI by adding `[.]` to the tags. Please do it because currently, your tests block our Windows test machines as they run into an endless loop.